src/main/java/calculatorTask/Calculator.java

```diff
@@ -0,0 +1,230 @@
+package calculatorTask;
+
+import java.util.*;
+
+public class Calculator{
+    public static HashMap<Character, Operation> operations = new HashMap<>();
```

Comment:
Reference object must be the interface while the object value - the implementation.
Thus: public static Map<Character, Operation> operations = new HashMap<>();

Reply:
Does this provide some functionality like methods that are only available for "Map" or is it just for cleaner code